app/pages/verificationwithpassword/verificationwithpassword.js

```diff
@@ -81,11 +88,15 @@ export let VerificationWithPassword = React.createClass({
   },
 
   render: function() {
+    if (_.get(this.props, 'notification.message', '') === errorMessages.ERR_BIRTHDAY_MISMATCH) {
+      this.props.trackMetric('VCA Home Verification - Birthday Mismatch')
+    }
```

Comment:
Hmm, tough one. Perhaps can we try doing it in `componentWillReceiveProps` and look carefully to see if it gets triggered more than once? I suppose we also might be able to do something like only call the metric if `this.props.notification.message === null && nextProps.notification.message === ERR_BIRTHDAY_MISMATCH` to avoid triggering except on initial prop change. Then we keep an eye on this type of situation - if an additional one or two things come along where making a metrics call directly within the action creators would be best, then I think we redo the metrics middleware to match the uploader's more closely so that the middleware looks for metrics in all actions and dispatches them as-is rather than dispatching a pre-configured metric for certain action types. Does that make sense?